src/main/java/interfaces/IHandlerRouter.java

```diff
@@ -1,6 +0,0 @@
-package com.td.HttpServer;
-
-public interface IHandlerRouter {
-  public Ihandler selectHandlerBadRequest();
```

Comment:
This file wasn't deleted. See comment above about it not being recognized by git as a rename.